sass/layout/_INDEX.scss

```diff
@@ -0,0 +1,16 @@
+@import "header";
+@import "hero";
+@import "best-way";
+@import "amazing-features";
+@import "conference-support";
+@import "why-appbox";
+@import "app-reviews";
+@import "appbox-screenshots";
+@import "more-apps";
+@import "download-app";
+@import "discounted-pricing";
+@import "team-members";
+@import "appbox-featured";
+@import "app-support";
+@import "footer";
+@import "flexContainer";
```

Comment:
I'm curious what the criteria was for what was a "component" vs. what was a "layout"? Where did you decide to draw that line, and why?